README.md

```diff
@@ -40,6 +38,29 @@ provisioning:
 ... and that you have a pre-configured `jessie` container that works, you should be able to get
 *something* out of `nomad` commands made in the same folder.
 
+You can also choose to define a container that will be created by pulling directly one of the image
+hosted on https://images.linuxcontainers.org/: this is the "pull" mode:
+
+```
+name: myproject
+image: ubuntu/xenial/amd64
```

Comment:
Yep. I totally agree. It turns out that I had a problem using both pull mode and "implicit" aliases like "ubuntu/xenial" (the container was not created and and error occurred). Using the "simplestreams" protocol instead of "lxd" seems to fix the problem.